### PR TITLE
Add setups for AWS S3 buckets

### PR DESCRIPTION
**Pull request by JAFAJ0**
### What this PR does (required):
- Set backend AWS-S3 controller and route for uploading files, and get Profile photo url---

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
-Not sure if I need to download files from S3 to display. If so, I gonna add this part when doing integrations.

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -21,14 +23,10 @@ function App(): JSX.Element {
           <AuthProvider>
             <SocketProvider>
               <Switch>
-                <Route exact path="/login" component={Login} />
+                <Route exact path="/profilePhoto" component={ProfilePhoto} />
```
[the remark this answers is not in the document]
Comment (rajivtitus) [reply client/src/App.tsx:27]: @JAFAJ0 Please address above comment before merging

File: client/src/App.tsx [hunk 2]
```diff
@@ -21,14 +23,10 @@ function App(): JSX.Element {
...
                 <Route exact path="/signup" component={Signup} />
-                <Route exact path="/dashboard">
-                  {/* <Dashboard /> disabled until Navbar setup */}
-                  <MySitters />
-                </Route>
                 <Route path="*">
-                  <Redirect to="/login" />
+                  <Redirect to="/profilePhoto" />
```
[the remark this answers is not in the document]
Comment (rajivtitus) [reply client/src/App.tsx:30]: @JAFAJ0 Please address above comment before merging

File: server/app.js [hunk 3]
```diff
@@ -49,12 +50,7 @@ app.use("/users", userRouter);
 app.use("/request", requestRouter);
 
 app.use("/profile", profileRouter);
-app.use("/availablity", availabilityRouter);
-
-app.use("/profile", profileRouter);
-app.use("/availablity", availabilityRouter);
-
-app.use("/profile", profileRouter);
+app.use("/profilephoto", photoRouter);
```
Comment (rajivtitus) [inline server/app.js:53]:
```suggestion
app.use("/profile-photo", photoRouter);
```

File: server/controllers/profilephoto.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    };
+    s3.upload(uploadParams, (err, data) => {
+        if (err) {
+            console.log('Error occured while trying to upload to S3 bucket', err);
+        }
+        if (data) {
+            fs.unlinkSync(req.file.path); // Empty temp folder
+            const locationUrl = data.Location;
+            profile.photoUrl = locationUrl;
+            newUser
```
Comment (rajivtitus) [inline server/controllers/profilephoto.js:50]: Where has this variable been defined?

Comment (JAFAJ0) [reply server/controllers/profilephoto.js:50]: Forget to edit this......

**Comment by JAFAJ0** [comment]
@rajivtitus Is this good to merge?

**Review by rajivtitus — APPROVED** [review]
@JAFAJ0 a couple of comments haven't been addressed, please address them prior to merging! Thank you :)

**Pull request merged**